### PR TITLE
Appium settings docs

### PR DESCRIPTION
**Pull request by dpgraham**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: commands-yml/commands/session/other/settings.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+endpoint:
+  url: /wd/hub/session/:session_id/appium/settings
+  method: 'POST'
+  url_parameters:
+    - name: session_id
+      description: ID of the session to route the command to
+  json_parameters:
+    - name: settings
+      type: string
+      description: Settings string
```
Comment (imurchie) [inline commands-yml/commands/session/other/settings.yml]: Do we have any indication of what this "Settings string" might be?

File: commands-yml/commands/session/other/update-settings.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+name: Get Device Settings
+short_description: Update the current setting on the device
```
Comment (imurchie) [inline commands-yml/commands/session/other/update-settings.yml]: Retrieve, not Update.

File: commands-yml/commands/session/other/update-settings.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+example_usage:
+  javascript_wd:
+    |
+      await driver.settings();
```
Comment (imurchie) [inline commands-yml/commands/session/other/update-settings.yml]: `let settings = driver.settings();`

File: commands-yml/commands/session/other/update-settings.yml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    xcuitest: true
+    uiautomation: true
+  android:
+    uiautomator2: true
+    uiautomator: true
+  mac:
+    mac: true
+  windows:
+    windows: true
+client_support:
```
Comment (imurchie) [inline commands-yml/commands/session/other/update-settings.yml]: If all these are true, why no examples?

**Review by imurchie — CHANGES_REQUESTED** [review]

File: commands-yml/commands/session/other/settings.yml (hunk 1 above)
Comment (dpgraham) [reply commands-yml/commands/session/other/settings.yml]: I'm not sure what that is tbh.

Comment (dpgraham) [reply commands-yml/commands/session/other/settings.yml]: Do you know anything about these commands @imurchie? It looks like they're not commonly used.

File: commands-yml/commands/session/other/update-settings.yml (hunk 4 above)
Comment (dpgraham) [reply commands-yml/commands/session/other/update-settings.yml]: @vmaxim @SrinivasanTarget do either of you know if this command is supported in Java client?

File: commands-yml/commands/session/other/settings.yml (hunk 1 above)
Comment (imurchie) [reply commands-yml/commands/session/other/settings.yml]: See https://github.com/appium/appium-base-driver/blob/master/lib/basedriver/device-settings.js

File: commands-yml/commands/session/other/update-settings.yml (hunk 4 above)
Comment (SrinivasanTarget) [reply commands-yml/commands/session/other/update-settings.yml]: yes it is supported @dpgraham

**Review by SrinivasanTarget — CHANGES_REQUESTED** [review]

**Pull request merged**